header_printer.go

```diff
@@ -66,19 +66,31 @@ func (p HeaderPrinter) Sprint(a ...interface{}) string {
 
 	text := Sprint(a...)
 
+	var overlappingCharactersBackgroundFiller string
 	if p.FullWidth {
-		p.Margin = (GetTerminalWidth() - len(text)) / 2
+		if (GetTerminalWidth()-len(text))/2 < 1 {
```

Comment:
`GetTerminalWidth()-len(text))/2` is reused on line 75 and can be stored in a variable instead of having to re-calculate the value.